docs/source/roles/role-edpm_ovn.rst

```diff
@@ -2,5 +2,10 @@
 Role - edpm_ovn
 ===========================
 
+Many of the parameters exposed by edpm_ovn use defaults derived
+values of other parameters. Take great care while overriding them,
+as changes may propagate further.
+
+
```

Comment:
We probably have a good opportunity to go through this role and see what might have made sense in tripleo, but doesn't quite make as much sense now. A lot of things were coming from Heat, they might now be easier to change.

Reply:
Yes. I've created a task for it.